### ISSUE TITLE
pam: pam_acct_mgmt failed: Permission denied

### ISSUE THREAD

**Issue by knamir** [rharmonson/rharmonson.github.io#9]
hi, i followed your lead and at first it all worked out. but now i get the following error message:  ERROR: (0) pam: pam_acct_mgmt failed: Permission denied
, but i don't understand what could be causing it. Help would be welcome. Thanks

**Comment by rharmonson** [comment]
Which article on rharmonson.github.io is this issue in reference to?

**Comment by knamir** [comment]
you are right. refers to the 2FA system with google authentication, freeradius, active directory and centos7

**Comment by rharmonson** [comment]
Not seen that one.

I would suggest starting with the man page for [pam_acct_mgmt](https://linux.die.net/man/3/pam_acct_mgmt) and enable debuggin [debugging_pam_config.pdf](https://github.com/rharmonson/rharmonson.github.io/files/5520617/debugging_pam_config.pdf). The man page is not very useful, but debugging may lead you to identifying the problem.
